File: warehouse-system/src/main/java/com/who/warehousesystem/dto/DpDetailsDto.java
package com.who.warehousesystem.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class DpDetailsDto {

    private Integer id;
    private String enName;
    private String arName;
    private LocalDate dDate;
}

File: WarehouseSystem/WarehouseSystem/Dto/DpDetailsDto.cs
using Newtonsoft.Json;

namespace WarehouseSystem
{
    class DpDetailsDto
    {
        public int id { get; set; }
        public string enName { get; set; }
        public string arName { get; set; }
        public string dDate { get; set; }

        public DpDetailsDto (int id,string enName,string arName,string dDate)
        {
            this.id = id;
            this.enName = enName;
            this.arName = arName;
            this.dDate = dDate;
        }

        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}
